opentelemetry-distro/src/opentelemetry/distro/__init__.py

```diff
@@ -34,5 +34,5 @@ class OpenTelemetryDistro(BaseDistro):
 
     # pylint: disable=no-self-use
     def _configure(self, **kwargs):
-        os.environ.setdefault(OTEL_TRACES_EXPORTER, "otlp_proto_grpc")
-        os.environ.setdefault(OTEL_METRICS_EXPORTER, "otlp_proto_grpc")
+        os.environ.setdefault(OTEL_TRACES_EXPORTER, "otlp")
+        os.environ.setdefault(OTEL_METRICS_EXPORTER, "otlp")
```

Comment:
Although the default protocol is gRPC we can be explicit about it here.

Reply:
If we use the current `otlp_proto_grpc`, in order to use HTTP users will have to configure both `TRACES_EXPORTER` and `OTEL_EXPORTER_OTLP_PROTOCOL` vars, which can be confusing.

Reply:
I was talking about adding `OTEL_EXPORTER_OTLP_{TRACES,METRICS}_PROTOCOL`.

Reply:
I see now.
Added `OTEL_EXPORTER_OTLP_PROTOCOL=grpc`, otherwise if we set a specific var the user will have to override it and won't be able to set the global var only